### ISSUE TITLE
unable to save last file before saving preceding files

### ISSUE THREAD

**Issue by xxjapp** [microsoft/vscode#86015]
- Version: 1.40.2 (user setup)
- Commit: f359dd69833dd8800b54d458f6d37ab7c78df520
- Date: 2019-11-25T14:54:45.096Z
- Electron: 6.1.5
- Chrome: 76.0.3809.146
- Node.js: 12.4.0
- V8: 7.6.303.31-electron.0
- OS: Windows_NT x64 10.0.17763

Steps to Reproduce:

1. create a new file and input something
2. create another new file and input something
3. try to save last file

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Not a regression, leaving to @bpasero once he comes back from vacation.
Also on insiders we have changed the save behavior, so this might be different now.

**Comment by bpasero** [comment]
@xxjapp how exactly do you do that?

> try to save last file

**Comment by xxjapp** [comment]
@bpasero 

to save file by "Ctrl + S" shortcut.

Now I noticed that in Menu the "Ctrl + S" shortcut is binded to "Save All". When I select "Save" (which has no shortcut binding). The save action acts as expected.

And Now I noticed that this "Ctrl +S" to "Save All" shortcut binding is defined by User(me maybe). When I deleted this binding, The "Crtl + S" works as expected.

Sorry for this issue. I think there is no problem.
